composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.5.0",
         "guzzlehttp/guzzle": "~6.0",
-        "symfony/console": "^2.5 || ^3.0"
+        "symfony/console": "~4.0"
```

Comment:
This is too restricting, `"^2.5 || ^3.0 || ^4.0"` would be preferred. This is changed in 7b337d234d37fde09017e6d502551f0022ba93fb